sandbox/registration-processor-dmz.properties

```diff
@@ -164,6 +164,10 @@ registration.processor.max.retry=3
 # for search API response consumed by regclient
 mosip.registration.processor.registration.status.external-statuses-to-consider-processed=UIN_GENERATED,REREGISTER,REJECTED,REPROCESS_FAILED	
 
+mosip.registration.processor.lostrid.version=1.0
+mosip.registration.processor.postalcode.req.url=https://dev.mosip.net/v1/masterdata/registrationcenters
```

Comment:
for this add a comment describing what is this property used for

Reply:
taken care